app/assets/stylesheets/layout_components/card.scss

```diff
@@ -0,0 +1,41 @@
+.card {
+  background: $white;
+  padding: $base-padding 0;
+  text-align: center;
+  margin-bottom: $base-margin;
+
+  &__title {
+    font-size: $base-font-size*1.45;
+    padding: 7px;
+    font-weight: bold;
+    text-align: center;
+
+    @include breakpoint(small only) {
+      font-size: $base-font-size*1.25;
+    }
+  }
+
+  &--blue-bg {
+    @extend .card__title;
+    background: $brand-blue-light;
+    color: $white;
+    position: relative;
+  }
+
+  &__image {
+    margin-bottom: $base-margin;
+  }
+
+  &__content-bg {
+    background: $ash;
+    border-top: 2px solid $stone;
+    border-bottom: 2px solid $stone;
+    padding: $base-padding 0;
```

Comment:
will having padding 0 left and right cause the content to go right up to the edge on smaller screens/if there is a lot of content? can we just deal with left and right padding by using columns?

Reply:
no, this gets the grey background to go edge to edge, but the content itself has it's own padding applied to it so it never touches the edges.

Reply:
does it still work if you just declare top and bottom padding (not setting left and right to 0)? you shouldn't have to be overriding anything with the 0?

Reply:
ya, now that I look at it again, I actually only need the padding-top. I think something else was applying it that maybe i've already removed from another comment on this review possibly.